src/raptor/RaptorTimeRangeQuery.ts

```diff
@@ -33,8 +34,7 @@ export class RaptorTimeRangeQuery<T> {
     const midnight = this.getMidnight(startRange);
     const startSeconds = (startRange.valueOf() - midnight.valueOf()) / 1000;
     const endSeconds = (endRange.valueOf() - midnight.valueOf()) / 1000;
-    console.log("Searching from ", startSeconds);
-    console.log("until ", endSeconds);
+    logger.debug("Searching from %o seconds until %o seconds", startSeconds, endSeconds);
```

Comment:
The string format should be a bit more precise I think. `startSeconds` and `endSeconds` aren't objects, they are just numbers at this point.

Reply:
8c6b938
--


 Changed type in log message string from object to number.